vignettes/SEIR_Model_Compartments.Rmd

```diff
@@ -0,0 +1,137 @@
+---
+title: "SEIaImIsR model"
+author: "Liangti Dai and David Seiferth"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{SEIaImIsR model}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(comomodels)
+library(ggplot2)
+library(glue)
+```
+
+## Introduction
+This document explains the basics of the SEIaImIsR model. The model describes
+how populations of susceptible ($S$), exposed ($E$), infectious and recovered ($R$)
+individuals evolve over time. The infected population is split into different 
+symptom compartments: asymptomatic ($I_a$), mild ($I_m$) and severe ($I_s$). 
+An ordinary differential equation (ODE) is defined for each of the populations.
+
+The object class `SEIaImIsR` lets the user define and simulate such a model, 
+choosing the initial conditions and parameter values.
+
+## The SEIaImIsR Model
+The SEIaImIsR model consists of six ODEs describing the six populations. Susceptible
+individuals ($S$) have not been exposed to the virus. Exposed individuals ($E$) have been
+exposed to the virus and have been infected but are not yet infectious to others; they are 
+in the incubation period. Infectious individuals can spread the virus to others. 
+The total initial population size is normalised to 1.
+The current model does not include natural death or birth.	
+The ODEs describing how these six populations evolve over time are
+$$\frac{\text{d}S}{\text{d}t} = -\lambda S (I_{\text{a}} + I_{\text{m}} + I_{\text{s}}) + \omega R,$$
+where $\lamda$ denotes the rate, at which an infected individual exposes susceptible individuals,
+and $\omega$ denotes the rate, at which recovered individuals become susceptible.
+$$\frac{\text{d}E}{\text{d}t} = \lambda S (I_{\text{a}} + I_{\text{m}} + I_{\text{s}}) -\gamma E,$$
+where $\gamma$ denotes the rate at which exposed individuals become infected.
+$$\frac{\text{d}I_{\text{a}}}{\text{d}t} = (1-\alpha_{E->I_m} -\alpha_{E->I_s}) \gamma E - \beta_{I_a->R}  I_{\text{a}} - \mu_{I_a->D} * I_{a},$$
```

Comment:
I'd be tempted to have an alpha_{E->I_a} since there's always talk of the proportion of individuals who are asymptomatic; there's also talk of the hospitalisation rate of patients (i.e. those becoming severely infected), so I'd have a alpha_{E->I_s}. So, I'd be tempted to drop alpha_{E->I_m}.